host/xtest/regression_1000.c

```diff
@@ -182,7 +182,7 @@ static void xtest_crypto_test(struct xtest_crypto_session *cs)
 	Do_ADBG_EndSubCase(cs->c, "SHA-256 test, 3 bytes input");
 
 	Do_ADBG_BeginSubCase(cs->c,
-			     "AES-256 ECB encrypt test, 32 bytes input, with fixed key");
+		"AES-256 ECB encrypt test, 32 bytes input, with fixed key");
```

Comment:
Before this string was aligned with the `(`, now I don't know what it's aligned with.

Reply:
ok, i will go another way.